conn.go

```diff
@@ -166,9 +166,15 @@ func (s *Swarm) addConn(netConn net.Conn, isServer bool) (*Conn, error) {
 	// unlocked **before** the Handler is called (which needs to be
 	// sequential). This was the simplest thing :)
 	setupConn := func() (*Conn, error) {
+
+		// create a new spdystream connection
+		ssConn, err := s.transport.NewConn(netConn, isServer)
+		if err != nil {
+			return nil, err
+		}
+
```

Comment:
oh interesting. setting up a connection _could_ be expensive (like RTT expensive). what about something like: a32d27b276851563ddfca718c012d1049e3155ae